integreat_cms/cms/views/pois/poi_form_view.py

```diff
@@ -70,6 +71,7 @@ def get(self, request, *args, **kwargs):
         poi_translation_form = POITranslationForm(
             instance=poi_translation, disabled=disabled
         )
+        url_link = f"{settings.WEBAPP_URL}/{region.slug}/{language.slug}/locations/"
```

Comment:
```suggestion
        url_link = f"{settings.WEBAPP_URL}/{region.slug}/{language.slug}/{poi_translation_form.instance.url_infix}/"
```
Instead of hardcoding the url part here, could you adapt the [url_infix](https://github.com/digitalfabrik/integreat-cms/blob/6a4be17ff0562b7c57e75c09cfce7af6064bc688/integreat_cms/cms/models/pois/poi_translation.py#L62) to `locations`?

Also, could you add the same context variable to the POST-view (in case the form is rendered with errors)...